zerver/webhooks/github/tests.py

```diff
@@ -146,7 +146,7 @@ def test_issue_comment_msg_with_custom_topic_in_url(self) -> None:
         self.check_webhook("issue_comment", expected_topic, expected_message)
 
     def test_issue_comment_pull_request_comment_msg(self) -> None:
-        expected_topic = "testing-gh / issue #23 Update first.txt"
+        expected_topic = "testing-gh / PR #23 Update first.txt"
```

Comment:
It's hard to read that this topic is same as the PR created topic. May be we can change the fixture so that the topic matches the global `TOPIC_PR`?

Reply:
Updated the new test fixture so that the repo name and issue number/title match the ones used in the other test fixtures used for PRs.